src/components/InLineNotification.tsx

```diff
@@ -68,7 +68,7 @@ export function InLineNotification({
         {!hideIcon && (
           <View style={styles.iconContainer}>
             {customIcon ?? (
-              <Icon color={variantColor.primary} size={20} testId="InLineNotification/Icon" />
+              <Icon color={variantColor.primary} size={16} testId="InLineNotification/Icon" />
```

Comment:
i think the icon size should always be 16 according to the [design system?](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=493-4&mode=design&t=mb5pNPoDPqSiXIKZ-0)

Reply:
ah, that's a good question!

some images are indeed 16, but some apparently are not.
in this [figma components library](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=311-3&mode=design&t=VzDUgt4lZH6YtD6z-0) every icon with the surrounding padding is 24 no matter of the actual image size.
notice that in notification designs the icons also have this padding around them.

my guess if we consider icons being 24 as a default approach, it should fit the designs nicely

Reply:
i reverted this change, i figure it's something we can address holistically later